Maquetacion/index.html

```diff
@@ -6,8 +6,16 @@
     <meta name="description" content="training">
     <meta name="author" content="Wolox">
     <link rel="stylesheet"  type="text/css" href="application.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   </head>
   <body>
     <h1 class="title">Front-End Tools</h1>
-  </body>
+    <div class="tech">
+      <img class="logo" src="assets/react_icon.svg" alt="">
+      <div>
+        <h2 class="text title-card">ReactJS</h2>
+        <p class="text age">Release:2013</p>
```

Comment:
Add a space between the colon and the date. `Release: 2013` instead of `Release:2013`.

Reply:
👍